library/std/build.rs

```diff
@@ -4,10 +4,8 @@ fn main() {
     println!("cargo:rerun-if-changed=build.rs");
     let target = env::var("TARGET").expect("TARGET was not set");
     if target.contains("freebsd") {
-        if env::var("RUST_STD_FREEBSD_12_ABI").is_ok() {
-            println!("cargo:rustc-cfg=freebsd12");
-        } else if env::var("RUST_STD_FREEBSD_13_ABI").is_ok() {
-            println!("cargo:rustc-cfg=freebsd12");
+        println!("cargo:rustc-cfg=freebsd12");
```

Comment:
With this minimum, we should drop the cfg altogether and remove `cfg(not(freebsd12))` code.

Reply:
AFAICS just a couple functions in `library/std/src/os/freebsd/fs.rs` are using that.